tests/tests_old/test_services.py

```diff
@@ -81,7 +81,7 @@
 
 NUM_DEVS_BEFORE = 3  # HGI, faked THM, faked REM
 NUM_DEVS_AFTER = 15  # proxy for success of cast_packets_to_rf()
-NUM_SVCS_AFTER = 10  # proxy for success
```

Comment:
This is to be expected, since some tests compare to a snapshot to check remotes, and some checks count something and compare before/after